README.md

```diff
@@ -6,3 +6,6 @@ The metadata is splitted in 3 classes:
   - Lengths
   - Patterns
   - Formats
+
+How to contribute:
```

Comment:
Thanks, would it be possible to add more lines here ?

Reply:
@sumitsharansatsangi would that be possible since it is fresh in your mind ?

Reply:
Yeah, sure.